gemonlinedb/src/common/FileUtils.cc

```diff
@@ -0,0 +1,34 @@
+#include "gem/onlinedb/detail/FileUtils.h"
```

Comment:
move under `src/common/detail/`

Reply:
Such a structure doesn't exist for other `detail` classes. Changing this is probably out of scope here (can be done as another PR to be merged before).